cmd/kubeadm/app/phases/kubelet/kubelet.go

```diff
@@ -18,13 +18,36 @@ package kubelet
 
 import (
 	"fmt"
+	"io"
+	"text/template"
 
+	"github.com/lithammer/dedent"
 	"k8s.io/klog/v2"
 
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	"k8s.io/kubernetes/cmd/kubeadm/app/util/initsystem"
 )
 
+var (
+	kubeletFailMsg = dedent.Dedent(`
+	Unfortunately, an error has occurred, likely caused by:
+		- The kubelet is not running
+		- The kubelet is unhealthy due to a misconfiguration of the node in some way (required cgroups disabled)
+
+	If you are on a systemd-powered system, you can try to troubleshoot the error with the following commands:
+		- 'systemctl status kubelet'
+		- 'journalctl -xeu kubelet'`)
+
+	controlPlaneFailTempl = template.Must(template.New("init").Parse(dedent.Dedent(`
+	Additionally, a control plane component may have crashed or exited when started by the container runtime.
```

Comment:
leaving the text like that even if the wiatforcontrolplanecomponents FG is disabled, still makes sense.
the error will actually say that it couldn't probe kub-apiserver at healthz for n minutes.